src/components/sidebar/routePathView/routePathInfoTab/RoutePathForm.tsx

```diff
@@ -125,15 +125,19 @@ class RoutePathForm extends React.Component<IRoutePathFormProps>{
                 <div className={s.flexRow}>
                     <InputContainer
                         label='VOIM. AST'
-                        value={Moment(routePath.startTime)
-                            .format(datetimeStringDisplayFormat)}
+                        type='date'
+                        value={routePath.startTime}
+                        onChange={this.onChange('startTime')}
                         disabled={isEditingDisabled}
+                        validatorRule={routePathValidationModel.date}
```

Comment:
Should we have routePathValidationModel validation fields with the same name as in IRoutePath?
routePathValidationModel.startTime
routePathValidationModel.endTime